Doc/library/fractions.rst

```diff
@@ -117,6 +117,12 @@ another rational number, or from a string.
 
       .. versionadded:: 3.8
 
+   .. method:: is_integer()
+
+      Return True if the Fraction is an integer.
```

Comment:
Nitpick: suggest inline code markup for `True`:
```suggestion
      Return ``True`` if the Fraction is an integer.
```

(We seem to be rather inconsistent about this across the codebase. The most authoritative source I could find is in the dev guide: https://devguide.python.org/documentation/markup/#quick-reference.)